components/Branding/Logo/Logo.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable max-len */
+import { bool } from 'prop-types';
+import styles from './Logo.module.css';
+
+export const Logo = ({ darkMode, markOnly, operationFill, codeFill, starFill, medalFill }) => {
```

Comment:
In the other PR, I said:

> You made a really reusable component here. I wanted to suggest simplifying and restricting it to just one prop variant and you can name them whatever you want. The API will be a lot lighter and less reusable, and I think that's okay because:
> 
> It's less code
> We can be strict about how we approve of our SVG being colored.
>

Reply:
I don't know if its less code but I'm on board for the open/closed principle here. Will make a Variants class that can be extended to more colors and encapsulate the default ones.